backend/main.py

```
@@ -1745,6 +1952,316 @@ async def get_active_tasks():
+        finetunes=active_finetunes,
+    )
+
+
+# ============================================
+# FINETUNE ENDPOINTS
+# ============================================
+
+@app.post("/profiles/{profile_id}/finetune/samples", response_model=models.FinetuneSampleResponse)
+async def add_finetune_sample(
+    profile_id: str,
+    file: UploadFile = File(...),
```

Ruff (0.15.2)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)